configure.ac

```diff
@@ -17,6 +17,7 @@ AC_PROG_CC
 AC_PROG_CC_STDC
 AC_PROG_MAKE_SET
 AM_PROG_AR
+AC_CHECK_PROG(MPICC_FOUND, [mpicc], "yes")
```

Comment:
While you are here, given the actual plugin does not require MPI and only the internal tests do, might I suggest adding a configure option to disable the test binary?  This allows libnccl-net.so to be built and packaged in an environment where MPI is not available.

Reply:
I am not necessarily an autotools guru, but this seems ok.